source/_docs/environment-specific-config.md

```diff
@@ -9,11 +9,36 @@ To quickly see which environment you are on, consider installing the [Pantheon H
 
 ## Customizations to wp-config.php
 
-Pantheon's default version of `wp-config.php` contains a nice example that sets `DISALLOW_FILE_MODS` to `true` on Test and Live environments.
-<script src="//gist-it.appspot.com/https://github.com/pantheon-systems/wordpress/blob/master/wp-config.php?footer=minimal&slice=88:91"></script>
+Pantheon's default version of `wp-config.php` demonstrates how to [set `DISALLOW_FILE_MODS` to `true` on Test and Live environments](https://github.com/pantheon-systems/WordPress/blob/master/wp-config.php#L88-L91){.external}:
 
-This is a useful model to follow for another recommended modification, defining `'WP_DEBUG', true` on all environments except for Test and Live.
-<script src="//gist-it.appspot.com/https://github.com/pantheon-systems/pantheon-settings-examples/blob/master/wordpress/wp_debug_dev.wp-config.php?footer=minimal"></script>
+```php
+if ( in_array( $_ENV['PANTHEON_ENVIRONMENT'], array( 'test', 'live' ) ) && ! defined( 'DISALLOW_FILE_MODS' ) ) :
+    define( 'DISALLOW_FILE_MODS', true );
+endif;
+```
+
+In the same way, you can conditionally set `'WP_DEBUG', true` based on the given Pantheon environment. For example, the following configuration enables WP_DEBUG for development environments (Dev and Multidevs), while disabling it on production environments (Test and Live): 
+
+```php
+/**
+ * For developers: WordPress debugging mode.
+ *
+ * Sets WP_DEBUG to true on if on a development environment.
+ *
+ */
+if (!defined('WP_DEBUG')) {
+    if(in_array( $_ENV['PANTHEON_ENVIRONMENT'], array('test', 'live'))) {
+      define('WP_DEBUG', false);
+    } else {
+      define( 'WP_DEBUG', true );
+    }
+}
```

Comment:
I'd like to get another technical review on this snippet to answer:  

1) Do we need to check the var is defined before calling it? Using `isset($_ENV['PANTHEON_ENVIRONMENT'])` -- otherwise this would cause a fatal error if they weren't on Pantheon (I think..) 
2) Do we need to check for command line condition, via `(php_sapi_name() != "cli")` ?

In context of the changes, I'm not sure if number 2 is a big deal but number 1 seems like a required standard we should adhere to any time we recommend conditional configurations like this

Reply:
@dustinleblanc I'm tagging you in on this since this is mostly your handiwork.

The only other use of `isset($_ENV['PANTHEON_ENVIRONMENT'])` is in an `else` statement that happens when we know it is set, but we know that by checking to see if it's _not_ set (`!isset($_ENV['PANTHEON_ENVIRONMENT'])`).

Reply:
@rachelwhitton @alexfornuto I think 29 as 
```php
if (!defined('WP_DEBUG') && isset($_ENV['PANTHEON_ENVIRONMENT'])) {
``` 
is probably the way to go?

Reply:
@dustinleblanc does moving the check for `PANTHEON_ENVIRONMENT` up into the main `if` statement protect from fatal errors if it isn't defined?